cmd/app/uninstall_test.go

```diff
@@ -48,7 +48,7 @@ func TestAppsUninstall(t *testing.T) {
 	testutil.TableTestCommand(t, testutil.CommandTests{
 		"Successfully uninstall": {
 			Setup: func(t *testing.T, ctx context.Context, clientsMock *shared.ClientsMock, clients *shared.ClientFactory) {
-				prepareCommonUninstallMocks(clients, clientsMock)
+				prepareCommonUninstallMocks(ctx, clients, clientsMock)
```

Comment:
note: This was missed in the PR that updated `.Setup: func(t, ctx, ...)` to accept the mocked context.